client/app/constants/app_constants.coffee

```diff
@@ -111,6 +114,8 @@ module.exports =
         'TOASTS_SHOW'               : 'TOASTS_SHOW'
         'TOASTS_HIDE'               : 'TOASTS_HIDE'
 
+        # Test
+        'RESET_FOR_TESTS'               : 'RESET_FOR_TESTS'
```

Comment:
I'm not sure that creating specific events for test is a good practice; Does any other event able to replace it?